api.md

```diff
@@ -1874,6 +1877,83 @@ are meant to implement data retention or perform tasks that will improve query
 performance on older chunks. Each policy is assigned a scheduled job
 which will be run in the background to enforce it.
 
+## add_job() :community_function: [](add_job)
+
+Register a custom procedure or function to be scheduled by our automation framework.
+
+#### Required Arguments [](add_job-required-arguments)
+
+|Name|Description|
+|---|---|
+| `proc` | (REGPROC) Name of the function or procedure to register as job|
+| `schedule_interval` | (INTERVAL) Interval between executions of this job|
+
+#### Optional Arguments [](add_job-optional-arguments)
+
+|Name|Description|
+|---|---|
+| `config` | (JSONB) Job-specific configuration (this will be passed to the function when executed)|
+| `initial_start` | (TIMESTAMPTZ) Time of first execution of job. |
```

Comment:
Minor nit:  Our standard is not to include "periods" at the end of the descriptions (unless there is a multiple sentence description).  This uses a mix, cut drop the period?

Reply:
OK that's not applied consistently then